cshsms/settings.py

```diff
@@ -16,7 +16,7 @@
 
 
 if not os.getenv('IS_TRAVIS', False):
-    from cshsms.settings_secret import SECRET_KEY, DATABASES, REMOTE
+    from cshsms.settings_secret import SECRET_KEY, DATABASES, REMOTE, TEXTLOCAL_API, TEXTLOCAL_PRIMARY_ID
 else:
     from cshsms.settings_travis import SECRET_KEY, DATABASES, REMOTE
```

Comment:
We should also import `TEXTLOCAL_API` and `TEXTLOCAL_PRIMARY_ID` from `settings_travis`.